### PR TITLE
[RFR] Further docs tweaking

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

File: docs/getting-started/about.rst [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ===============
 About PyMedPhys
-===============
\ No newline at end of file
+===============
+
+PyMedPhys is a toolbox for Medical Physicists that we all have access to, and
+we will all have access to whatever it becomes in the future.
```
Comment (Centrus007) [inline docs/getting-started/about.rst]: Sorry, I know you asked me to draft something. I found it very difficult to start (and not simply regurgitate the README, so I left it for further thought/discussion. :)

This is a perfectly suitable start!

File: README.rst [hunk 2]
```diff
@@ -38,31 +49,37 @@ Our Team
 
 PyMedPhys is what it is today due to its maintainers and developers. The
 currently active developers and maintainers of PyMedPhys are given below
-in alphabetical order along with their affiliation:
```
Comment (SimonBiggs) [inline README.rst]: Matt asked that I remove the alphabetical idea. We'll see how we go with this list into the future.

File: docs/getting-started/about.rst (hunk 1 above)
Comment (SimonBiggs) [reply docs/getting-started/about.rst]: In the end I imported just the description and team from readme :)

Comment (SimonBiggs) [reply docs/getting-started/about.rst]: No need to be sorry :)

File: docs/getting-started/about.rst [hunk 3]
```diff
@@ -1,3 +1,7 @@
 ===============
 About PyMedPhys
-===============
\ No newline at end of file
+===============
+
+.. include:: ../../README.rst
+   :start-after: .. START_OF_ABOUT_IMPORT
+   :end-before: .. END_OF_ABOUT_IMPORT
```
Comment (SimonBiggs) [inline docs/getting-started/about.rst]: This is something quite neat about reST that I learnt tonight :)

**Comment by SimonBiggs** [comment]
This one is ready for review and then merge once you're happy with it :)

File: README.rst [hunk 4]
```diff
@@ -122,9 +142,13 @@ hold:
     any and all other commercial damages or losses), even if such Contributor
     has been advised of the possibility of such damages.
 
-For the definitions of where the definition of License is taken to be the AGPLv3+
-with additional terms from the Apache 2.0. The definitions of Contributor, You,
-and Work for this clause are as defined within the Apache 2.0 license.
+For the definitions of where the definition of License is taken to be the
+AGPLv3+ with additional terms from the Apache 2.0. The definitions of
```
Comment (Centrus007) [inline README.rst]: Don't think this sentence makes sense...

**Comment by SimonBiggs** [comment]
Are you able to give that bit a shot? I've packed it in for the night.

File: changelog.md [hunk 5]
```diff
@@ -44,6 +44,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
   within Excel
 - Added rudimentary code to pull profiles from Mephysto
   files.
+- The previously separate `decodetrf` library is now distributed as part of
+  `PyMedPhys`. In particular, now by installing `PyMedPhys` alone one can run
+  `pymedphys trf to-csv` within the command line to convert `.trf` files into
+  `.csv` files.
+
```
Comment (Centrus007) [inline changelog.md]: Maybe reword to:
```
- The previously separate `decodetrf` library is now distributed within
  PyMedPhys. You can now simply install PyMedPhys and run
  `pymedphys trf to-csv` within the command line to convert `.trf` files into
  `.csv` files.
```

**Comment by SimonBiggs** [comment]
Feel free to use this branch. I'm done with it.

File: docs/getting-started/about.rst (hunk 3 above)
Comment (Centrus007) [reply docs/getting-started/about.rst]: Very nice!

File: docs/getting-started/roadmap.rst [hunk 6]
```diff
@@ -1,3 +1,25 @@
...
+=======
+
+Here will be recorded the roadmap for the future development of PyMedPhys.
+
+
+Graphical User Interface (GUI)
+------------------------------
+
+A goal of the graphical user interface is to have it customisable, and
+expandable. It is envisioned that the not yet released
```
Comment (Centrus007) [inline docs/getting-started/roadmap.rst]: I've noticed this style of writing quite a few times as I've perused the docs. Active verbs form more efficient, powerful statements. They hold interest. Phrases like:

"**We envision** that the..."

should be preferred over

"**It is envisioned** that the..."

Also, this isn't an academic paper, *don't be afraid to use pronouns*. Heck, business/academia's over-insistence on avoiding them in favour of "it" constructions is a travesty of quality English prose.

File: docs/getting-started/roadmap.rst [hunk 7]
```diff
@@ -1,3 +1,25 @@
...
+`JupyterLab Dashboarding`_ might provide a perfect environment for building
+GUIs that can be easily extended by the community.
+
+To progress in this direction we need to begin building `Jupyter Widgets`_ that
+would become the dashboarding building blocks. An example of one of these
```
Comment (Centrus007) [inline docs/getting-started/roadmap.rst]: "**An** example of **one** of these [...] **are** the..."

File: README.rst (hunk 2 above)
Comment (Centrus007) [reply README.rst]: No worries

File: docs/getting-started/roadmap.rst (hunk 6 above)
Comment (SimonBiggs) [reply docs/getting-started/roadmap.rst]: :) I think I've been tainted. I'm too stuck in not being able to use pronouns.

File: README.rst (hunk 4 above)
Comment (SimonBiggs) [reply README.rst]: Is this better? I was rewording that a little on the fly and I just left the part of the question in there.

**Comment by SimonBiggs** [comment]
Once you're happy with that reword I just did, what do you say we merge this one in, and cut a release today?

**Comment by Centrus007** [comment]
@SimonBiggs, yep, perfect!

I figured as much, I do exactly that all the time.

I was reluctant to edit it myself because it was entering into legal territory. I didn't want to even slightly misrepresent what you were trying to say.

**Comment by SimonBiggs** [comment]
> I was reluctant to edit it myself because it was entering into legal territory. I didn't want to even slightly misrepresent what you were trying to say.

Yup, that's fair :).
